site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -146,3 +146,6 @@ Supported DL models are not bound to the label and can be used for any objects.
 Fast online Object Tracking and Segmentation. Tracker is able to track different objects in one server request.
 Trackable object will be tracked automatically if the previous frame was
 a latest keyframe for the object. Have tracker indication on canvas. `SiamMask` tracker supported CUDA.
+
+> Note that `SiamMask` tracker is optimized for GPUs,
+> if you plan to use the CPU for tracking, use [TrackerMIL from OpenCV](/docs/manual/advanced/opencv-tools/#trackermil).
```

Comment:
```suggestion

> If you plan to track simple non-overlapping objects consider using fast client-side [TrackerMIL from OpenCV](/docs/manual/advanced/opencv-tools/#trackermil).
```
If we are talking about SiamMask compared to MIL, no matter where it is deployed it will be more accurate but slower. On simple objects their accuracy is almost the same, so faster tracking is more useful.